### ISSUE TITLE
Declare AdaptyErrorCode static variables as constant

### ISSUE THREAD

**Issue by jezsung** [adaptyteam/AdaptySDK-Flutter#110]
The `AdaptyErrorCode` variables are currently declared as `static final`.

This makes it impossible to use the error codes with the `switch` expression.

I suggest the following code changes to all static variables:
```dart
class AdaptyErrorCode {
  static final int unknown = 0;
  // to
  static const int unknown = 0;
...
```

**Comment by x401om** [comment]
Hi @jezsung! That seems to be a good enhancement. I think we will implement that in the next release. Thank you!

**Comment by x401om** [comment]
Hi @jezsung! Fixed in [3.2.1](https://github.com/adaptyteam/AdaptySDK-Flutter/releases/tag/3.2.1)
